CHANGELOG.next.toml

```diff
@@ -52,3 +52,9 @@ message = "Fix bug that can cause panics in paginators"
 references = ["smithy-rs#1903", "smithy-rs#1902"]
 meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client"}
 author = "rcoh"
+
+[[smithy-rs]]
+message = "Fix cargo audit issue on chrono."
+references = ["smithy-rs#1907"]
+meta = { "breaking" = false, "tada" = false, "bug" = false }
```

Comment:
You're missing the `target` key here.

Reply:
Thanks for catching! Addressed in eb68d7f